### PR TITLE
OIL-125: invoke added callback only in __cmp method

### PR DESCRIPTION
**Pull request by ltparis2018**

### CODE REVIEW

File: test/specs/core/core_oil.spec.js [hunk 1]
```diff
@@ -68,8 +68,7 @@ describe('core_oil', () => {
     }, 2000);
   });
 
-  // FIXME @alex can't fix it, pls help
-  xit('should not execute command collection and attach command collection execution to window object if optin is not provided', (done) => {
+  it('should not execute command collection and attach command collection execution to window object if optin is not provided', (done) => {
```
Comment (ltparis2018) [inline test/specs/core/core_oil.spec.js]: Hm, what's the problem here? Test works for me if activated.

Comment (CanardSauvage) [reply test/specs/core/core_oil.spec.js]: It didn't work when I changed the code last week ;) Maybe we fixed it with the whole lotta other stuff we changed the last days ;)

File: src/scripts/core/core_command_collection.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import { logError, logInfo } from './core_log';
-import { getCommandCollection } from './core_utils';
-import { getConsentDataString, getVendorConsentData } from './core_consents';
-import { getVendorList, loadVendorList } from './core_vendor_information';
+import {logError, logInfo} from './core_log';
```
Comment (CanardSauvage) [inline src/scripts/core/core_command_collection.js]: This is a conflict between your IDE and mine - we need to add this to the .editorconfig

**Comment by CanardSauvage** [comment]
Super nice Alex, well done!!!

**Pull request merged**